include/diy/decomposition.hpp

```diff
@@ -194,7 +194,7 @@ namespace detail
             AddBlock(diy::Master* master):
               master_(master)               {}
 
-    void    operator()(int gid, const Bounds& core, const Bounds& bounds, const Bounds& domain, const Link& link) const
+    void    operator()(int gid, const Bounds& /*core*/, const Bounds& /*bounds*/, const Bounds& /*domain*/, const Link& link) const
```

Comment:
You can just drop the unused parameters, although I don't feel too strongly about this.

Reply:
Done.